### PR TITLE
POC-122: Add Colour on Clinical Testing Build

### PR DESCRIPTION
**Pull request by hiqedme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ npm start
 
 Fire up your chrome and go to `https://localhost:3000`. You should see a login screen with the AMPATH logo.
 
+### Building for deployment
+
+When building for production environment use:
+`npm run build-prod`
+When building for staging or test environment use:
+`npm run build-staging`
+
```
Comment (kipchubett) [inline README.md:49]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -56,3 +75,4 @@ There are a few tools that we use extensively that all AMPATH developers should
 - AMRS is AMPATH's medical records system. You'll need an AMRS account for use with a lot of AMPATH's internal tools, including AMPATH POC.
 
 Access to these platforms is managed by the AMPATH IT team.
+```
```
Comment (kipchubett) [inline README.md]:
```suggestion
```

File: src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts [hunk 3]
```diff
@@ -37,12 +38,14 @@ export class PatientBannerComponent implements OnInit, OnDestroy, OnChanges {
...
   public attributes: any;
   public birthdate;
+
   public formattedPatientAge;
   private subscription: Subscription;
   private subs = [];
   private patientServiceSubscription: Subscription;
   public relationships: any = [];
   public relationship: Relationship;
+  public isNotProd = true;
```
Comment (kipchubett) [inline src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts]: No need for "not". Just imagine `!isNotProd`

**Review by kipchubett — CHANGES_REQUESTED** [review]
Thanks @hiqedme for working on this.
Can you squash the commits. Otherwise LGTM!

File: src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts [hunk 4]
```diff
@@ -37,12 +38,14 @@ export class PatientBannerComponent implements OnInit, OnDestroy, OnChanges {
   public searchIdentifiers: any;
   public attributes: any;
   public birthdate;
+
```
Comment (drizzentic) [inline src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts:41]:
```suggestion
```

File: src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts (hunk 3 above)
Comment (hiqedme) [reply src/app/patient-dashboard/common/patient-banner/patient-banner.component.ts]: Changed to `isStaging`

**Review by drizzentic — CHANGES_REQUESTED** [review]

**Review by kipchubett — APPROVED** [review]

**Review by drizzentic — APPROVED** [review]

**Pull request merged**